xtl.pc.in

```diff
@@ -0,0 +1,8 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/include
+
+Name: xtl
+Description: Basic tools (containers, algorithms) used by other quantstack packagesexpressions.
```

Comment:
Typo? `packagesexpressions`?

Reply:
You are right! I have opened a new PR that fixes it.